patch-package
--- a/node_modules/mobx-state-tree/dist/types/complex-types/model.d.ts
+++ b/node_modules/mobx-state-tree/dist/types/complex-types/model.d.ts
@@ -30,7 +30,7 @@ export declare class ModelType<S, T> extends ComplexType<S, T> implements IModel
         [K in keyof TP]: IType<any, TP[K]>;
     } & {
         [K in keyof SP]: IType<SP[K], any>;
-    }): IModelType<S & SP, T & TP>;
+    }): IModelType<S & SP & Snapshot<SP> , T & TP>;
     volatile<TP>(fn: (self: T) => TP): IModelType<S, T & TP>;
     instantiateVolatileState(self: T, state: Object): void;
     extend<A extends {
