patch-package
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -8739,7 +8739,7 @@ declare global {
         ignoredYellowBox: string[];
     }
 
-    const console: Console;
+    // const console: Console;
 
     /**
      * Navigator object for accessing location API
@@ -8747,10 +8747,11 @@ declare global {
      */
     interface Navigator {
         readonly product: string;
+        // @ts-ignore
         readonly geolocation: Geolocation;
     }
 
-    const navigator: Navigator;
+    // const navigator: Navigator;
 
     /**
      * This contains the non-native `XMLHttpRequest` object, which you can use if you want to route network requests
